### PR TITLE
feat: referendum timeline

### LINKED ISSUE #1790: OpenGov. Referendum. Additional
As a user I want to see additional info about referendum.

Todo:
- [x] Additional section with links
- [x] Timeline (from off-chain data source)
- [x] Advanced button
- [x] Advanced modal window with info
- [ ] Show identity for account if it's available

[Fima link](https://www.figma.com/design/ylvl7Ox25UuZj73Rn85DK7/OpenGov?node-id=574-100613&t=RafEPHCQCBylgBPp-4)
[Figma link to timeline](https://www.figma.com/design/ylvl7Ox25UuZj73Rn85DK7/OpenGov-voting?node-id=574-100548&t=hZlnKtCMb35nXJZl-4)
[Figma link to advanced popup](https://www.figma.com/design/ylvl7Ox25UuZj73Rn85DK7/OpenGov-voting?node-id=273-26846&t=hZlnKtCMb35nXJZl-4)

### PR DESCRIPTION
**Pull request by johnthecat**
Implemented referendum timeline card in details dialog 
Timeline

### CODE REVIEW

File: src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <div className="flex items-center justify-between">
+          <Shimmering height={18} width={120} />
+          <Shimmering height={18} width={80} />
+        </div>
+      )}
+
+      {!loading &&
+        timeline.map((status) => (
+          <div key={status.status} className="flex items-center justify-between">
+            <FootnoteText>{format(status.date, 'd MMM’yy, hh:mm')}</FootnoteText>
```
Comment (Asmadek) [inline src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx]: Better to use localized version for date, in order to not rewrite it in the future

File: src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    default:
+      return 'default' as const;
+  }
+};
+
+export const Timeline = ({ referendumId }: Props) => {
+  const { t } = useI18n();
+
+  const loading = useUnit(detailsAggregate.$isTimelinesLoading);
```
Comment (Asmadek) [inline src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx]: I think we should discuss naming for boolean, mostly we use `is` prefix

**Review by Asmadek — APPROVED** [review]

File: src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx (hunk 2 above)
Comment (johnthecat) [reply src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx]: Changed name to `isLoading` for now

File: src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx (hunk 1 above)
Comment (johnthecat) [reply src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx]: Current design requires this format, I think we should discuss this topic in context of i18n

**Review by sokolova-an — APPROVED** [review]

**Pull request merged**